traits/src/auction.rs

```diff
@@ -1,26 +1,47 @@
 use codec::FullCodec;
-use rstd::fmt::Debug;
-use sr_primitives::traits::MaybeSerializeDeserialize;
+use codec::{Decode, Encode};
+use rstd::{fmt::Debug, result};
+use sr_primitives::traits::{MaybeSerializeDeserialize, SimpleArithmetic};
 
+/// Auction info.
+#[cfg_attr(feature = "std", derive(Debug, PartialEq, Eq))]
+#[derive(Encode, Decode)]
```

Comment:
```suggestion
#[derive(Encode, Decode, RuntimeDebug)]
```
`use sr_primitives::RuntimeDebug`